dist/scripts/controllers/UserModalCtrl.js

```diff
@@ -0,0 +1,15 @@
+(function(){
+	function UserModalCtrl($scope, $uibModalInstance, $cookies){
+		$scope.confirm = function(username){
+			if (username){
+				$cookies.put('blocChatCurrentUser', username);
```

Comment:
This doesn't check for just whitespace, I'd suggest doing a `trim()` on the username and making sure it's not blank _before_ creating the cookie.